security/mobile-app-network-security.md

```diff
@@ -0,0 +1,126 @@
+# Abstract
+
+In this document we will discuss some of the common problems around network security for mobile developers consuming AeroGear services in their apps, and propose a solution that will help resolve those problems.
+
+We will first look at some of the use cases in order to define the problems, and then look at some of the options to solve them. Finally the proposed solution will be presented.
+
+# Goals
+
+## Primary Goal
+
+* Reach agreement on a solution that can help mobile developers properly handle network security in their mobile apps that consume AeroGear services
+
+## Secondary Goal
+
+* Provide use cases and try to explain the problems to audiences that may not be familiar with the area, to help them understand and maybe decide the business value of the solution
```

Comment:
This sounds like a goal of the PR rather than a goal of doing what the proposal is proposing.
Does that make sense?

Unless I'm misunderstanding.
Who is the 'audiences' in this goal?

Reply:
Not sure I get it. Is there a problem with this secondary goal, or do you think this proposal doesn't meet this goal?

The `audiences` here are anyone who is not technical or familiar with the area.

Reply:
Does reaching this goal mean that we've produced content (docs, videos, examples etc..) that try to explain the problems for people who aren't familiar with them?

Reply:
right, no. Both goals are only for this proposal. That's why there is a `problem explained` section in this proposal to try explain the problems in a bit more detail.

Reply:
Understood.
So I think usually the Goals of proposals are the outcome of doing what the proposal is suggesting, if you get what I mean.

e.g. 'Allow Mobile Developers using the Aerogear SDK to pin certificates for Aerogear Services being used'

with a secondary goal of maybe 'Allow Mobile Developers using the Aerogear SDK to pin certificates for custom services/runtimes'

I'm not saying these are the goals, just that the outcome of doing this proposal isn't completely clear to me

Reply:
Ok, that is not how I understand what are the "goals". For now, I will change the title to "Goals of the proposal" to make it clear

Reply:
perfect, thanks